src/main/java/com/app/katchup/MeetingResponse/MeetingInboxResponseController.java

```diff
@@ -26,31 +23,55 @@
     UserService userService;
 
     @PostMapping("/inbox")
-    public ResponseEntity<MeetingInboxResponse> postInbox(@RequestBody MeetingInboxResponse meetingInboxObject){
+    public ResponseEntity<MeetingInboxResponse> postInbox(@RequestBody MeetingInboxResponse meetingInboxObject) {
         MeetingInboxResponse invite = meetingResponseService.postInboxForUserName(meetingInboxObject);
         logger.info(String.format("Posted meeting invite having meeting id %s for user:%s", meetingInboxObject.getMeetingId(),
                 meetingInboxObject.getUserName()));
         return new ResponseEntity<>(invite, HttpStatus.OK);
     }
 
     @GetMapping("/inbox/{userName}")
-    public ResponseEntity<List<Inbox>> getInboxForUserName(@PathVariable String userName){
+    public ResponseEntity<List<Inbox>> getInboxForUserName(@PathVariable String userName) {
         //Returns only the meeting-id
         List<Inbox> invites = meetingResponseService.getInboxForUserName(userName);
         logger.info(String.format("Returning %s meeting invites for user:%s", invites.size(), userName));
         return new ResponseEntity<>(invites, HttpStatus.OK);
     }
 
+    @GetMapping("/meetings/{meetingId}/response")
+    public ResponseEntity<MeetingInboxResponse> getMeetingResponseForUser(@RequestBody MeetingRequestBody body) {
```

Comment:
```suggestion
    public ResponseEntity<MeetingInboxResponse> getMeetingResponseForUser(@RequestBody MeetingRequestBody body) {
```
Remove body and add it to header